CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-ledger ChangeLog
 
+## 11.0.1 - 2020-TBD
```

Comment:
this will be a breaking 12 release.

Reply:
Why? These changes are internal to the test suite.

Reply:
ah sorry, I'll update that. I thought it'd be a patch or minor change since the module itself doesn't depend on bedrock-identity and the changes were made in tests.

Reply:
@JSAssassin @dlongley aplogies. Surprised to learn that bedrock-ledger-node did not actually have a peer dep on bedrock-identity and this was only test suite related.  Carry on!  :+1: